CONTRIBUTING.md

```diff
@@ -209,14 +209,13 @@ Some of the BORS commands are as follows:
 
 ---
 
-## Regular Community Meeting
+## Monthly Community Meetings
+
+OpenEBS holds a monthly community meeting via Zoom on the last Thursday of the month, at 14:00 UTC.
+
+Meeting Link: https://us05web.zoom.us/j/87535654586?pwd=CigbXigJPn38USc6Vuzt7qSVFoO79X.1
+Starting in August 2024, the meetings will be recorded and posted on YouTube.
```

Comment:
Worthwhile to share the link for openebs YouTube channel?

Reply:
Sure, can do